app/src/main/java/com/example/tour_a_versity_v3/dto/LocationDTO.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.tour_a_versity_v3.dto
+
+import com.google.android.gms.maps.model.LatLng
+
+data class LocationDTO(var lat: Double, var lng: Double, var title: String, var snippet: String) {
+    fun getLatLng() = LatLng(lat, lng)
```

Comment:
A helper method that directly creates and returns a `LatLng` object for this location. It's not necessary, but if you find yourself doing this multiple times, this might be a worthy refactor. You can also make helper methods for other operations if you run into them.